app/models/application_form.rb

```diff
@@ -129,7 +129,7 @@ def equality_and_diversity_answers_provided?
     candidate.update!(candidate_api_updated_at: Time.zone.now) if form.changed.include?('phase') || created_at == updated_at
   end
 
-  after_commit :geocode_address_and_update_region_if_required
+  after_update :geocode_address_and_update_region_if_required
```

Comment:
I wonder if another way to do this would be to check if this is a `new_record?` and only geocode if false 🤔 Will check